assets/components/src/card/index.js

```diff
@@ -22,11 +22,12 @@ class Card extends Component {
 	 * Render
 	 */
 	render() {
-		const { buttonsCard, className, isSmall, noBorder, ...otherProps } = this.props;
+		const { buttonsCard, className, cptHeader, isSmall, noBorder, ...otherProps } = this.props;
 		const classes = classNames(
 			'newspack-card',
 			className,
 			buttonsCard && 'newspack-card__buttons-card',
+			cptHeader && 'newspack-card__cpt-header',
```

Comment:
Not sure about the naming here – it's not tied to the concept of a custom post type, and it's not used with CPTs exclusively (segments are not CPTs)

Reply:
True. Renamed `headerActions` in https://github.com/Automattic/newspack-plugin/pull/845/commits/39d164a53b0267dd5969ae87665db278ecec0cbf